src/iris/ui/static/js/iris.js

```diff
@@ -1890,16 +1891,23 @@ iris = {
       supportedTimezones: [],
       $page: $('.main'),
       $priority: $('#priority-table'),
+      $customNotificationTable: $('#custom-notification-table'),
```

Comment:
We should refer to user settings as "overrides" rather than "notifications". The current terminology is confusing, since we already use "notification" to refer to something else.